src/wp-includes/class-wp-block-bindings-registry.php

```diff
@@ -72,10 +72,10 @@ final class WP_Block_Bindings_Registry {
 	 *
 	 * @since 6.5.0
 	 *
-	 * @param string   $source_name       The name of the source. It must be a string containing a namespace prefix, i.e.
+	 * @param string $source_name       The name of the source. It must be a string containing a namespace prefix, i.e.
 	 *                                    `my-plugin/my-custom-source`. It must only contain lowercase alphanumeric
 	 *                                    characters, the forward slash `/` and dashes.
```

Comment:
These space should also adjust

Reply:
@mukeshpanchal27 Thank you for the suggestion, I have applied your suggestion in the latest commit.